src/Components/Login/login.css

```diff
@@ -1,13 +1,35 @@
+.title {
+    position: absolute;
+    top: 100px;
+    left: 46%;
+    margin: auto;
+    font-size: 20px;
+}
+
 .login {
     position: absolute;
-    top: 50%;
+    top: 45%;
     left: 35%;
     margin: auto;
     font-size: 20px;
 }
 
-input[type="text"] {
+input {
     width: 200%;
-    height: 18px;
+    height: 20px;
+    font-size: 14px;
```

Comment:
Het is beter om font-size met rem of em te defineren, dit zorgt ervoor dat alles beter schaalt met de gebruikers browserinstellingen. Zie; https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97